scripts/participateInAuction.js

```diff
@@ -1,19 +1,25 @@
 /* eslint-disable no-console */
 const Coordinator = artifacts.require("Coordinator")
+const { Gastimator } = require("./gas_station")
```

Comment:
nice abstraction, could even become its own node module at some later point.

Reply:
Yes that would be super cool! I would love to make something like that!